docs/general/builtins/window-functions.rst

```diff
@@ -514,11 +512,8 @@ Example::
    ...   budget,
    ...   LAG(budget) OVER(
    ...      PARTITION BY dept_id) prev_budget
-   ... FROM unnest(
-   ...   [1, 1, 2, 2, 2],
-   ...   [2017, 2018, 2017, 2018, 2019],
-   ...   [45000, 35000, 15000, 65000, 12000]
-   ... ) as t (dept_id, year, budget);
+   ... FROM (VALUES(1, 2017, 45000), (1, 2018, 35000), (2, 2017, 15000), (2, 2018, 65000), (2, 2019, 12000))
```

Comment:
Similarly, do you think it'd be more readable here if each row in the `VALUES` expression was on a new line?